README.md

```diff
@@ -8,7 +8,7 @@
 
 &nbsp;
 
-[![CI](https://github.com/iasql/iasql-engine/workflows/CI/badge.svg)](https://github.com/iasql/iasql-engine/actions?query=workflow%3ACI)
+[![CI Tests](https://github.com/iasql/iasql-engine/workflows/ci.yml/badge.svg)](https://github.com/iasql/iasql-engine/actions/workflows/ci.yml/badge.svg)
```

Comment:
hmm the first URL (https://github.com/iasql/iasql-engine/workflows/ci.yml/badge.svg) returns `Not found` for me. Maybe it should be:
```suggestion
[![CI Tests](https://github.com/iasql/iasql-engine/actions/workflows/ci.yml/badge.svg)](https://github.com/iasql/iasql-engine/actions/workflows/ci.yml)
```